pom.xml

```diff
@@ -412,163 +287,18 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-enforcer-plugin</artifactId>
-                <version>1.3.1</version>
```

Comment:
version is 1.4 in parent pom. Not necessarily a problem but might be worth checking?